### ISSUE TITLE
ShouldUpdate always return false

### ISSUE THREAD

**Issue by sipi** [bennypowers/lit-apollo#16]
https://github.com/bennypowers/lit-apollo/blob/7c1413a21ad66c05d2e8f1eab6596dd7d8bc69f2/apollo-query.js#L148-L165

I does not understand all the stuffs but I fixed the issue in my own project by replacing this method by 

```
shouldUpdate(changed, old) {
    return !!this.data || !!this.error || this.loading != null;
}
```
because data, error and loading are class properties and shouldUpdate() which is defined in "UpdatingElement" from @polymer/lit-element does not take a first parameter containing these data (it take a map called _changedProperties) so data, error and loading is always undefined…

**Comment by bennypowers** [comment]
Yes this is an error, thank you for finding it. I'll push out a new version shortly.

**Comment by sipi** [comment]
Nice :)

**Comment by bennypowers** [comment]
Published to npm as v0.6.8
